test/edu-benefits/1990-rjsf/config/militaryService.unit.spec.jsx

```diff
@@ -7,7 +7,7 @@ import ReactTestUtils from 'react-dom/test-utils';
 import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
 import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form';
 
-describe('Edu 1995 militaryHistory', () => {
+describe('Edu 1990-rjsf militaryService', () => {
```

Comment:
You can just use 1990, instead of 1990-rjsf, so we don't have to change this later.